app/modules/generate_cookies.py

```diff
@@ -130,10 +129,9 @@ def read_cookies_from_file(path:str='cookie_info.txt') -> dict:
             for line in f:
                 line = line.strip()  # Remove any surrounding whitespace or newline characters
                 if '=' in line:  # Ensure the line has the correct format
-                    name, value = line.split('=', 1)  # Split only at the first '='
+                    name, value = line.split('=', 0)  # Split only at the first '='
```

Comment:
_split(sep,  **Maximum number of splits**)_, this can't be 0 even though everything else in this industry starts with 0
```suggestion
                    name, value = line.split('=', 1)  # Split only at the first '='
```